packages/salesforce-adapter/src/transformers/xml_transformer.ts

```diff
@@ -611,7 +611,7 @@ export const createDeployPackage = (deleteBeforeUpdate?: boolean): DeployPackage
       const instanceName = await apiName(instance)
       const setAndLogZipFile = (fileName: string, content: string | Buffer): void => {
         zipContent.set(fileName, content)
-        log.trace('Added XML file %s with content %s', fileName, content)
+        log.trace('Added XML file %s', fileName)
```

Comment:
```suggestion
        log.debug('Added XML file %s', fileName)
```
Why are we removing this log though? This was the only way we could actually inspect the values we send to Salesforce. If we still choose to remove the content from this log, let's make it debug.

Reply:
Removing this log because `content` is often a buffer which breaks DD.
We could convert to string but in some cases it's a non-text file.